requirements.in

```diff
@@ -1,7 +1,9 @@
--e git://github.com/mbertheau/python-emarsys@master#egg=python-emarsys
+-e git://github.com/mbertheau/python-emarsys@0.3#egg=python-emarsys
+
 # lowest supported django version
-Django<1.8
+Django>1.8
```

Comment:
Unfortunately not. Only the Django 1.9 line is compatible with Python 3.5 which is being used in the signup service. Anything below 1.9 throws up during the test run.
```
  File ".../.tox/py35/lib/python3.5/site-packages/django/utils/html_parser.py", line 12, in <module>
    HTMLParseError = _html_parser.HTMLParseError
AttributeError: module 'html.parser' has no attribute 'HTMLParseError'
```

I changed the version requirements though to be a little bit more strict.